frontend/packages/kubevirt-plugin/src/selectors/vm/selectors.ts

```diff
@@ -151,3 +152,23 @@ export const getNodeSelector = (vm: VMKind) => vm?.spec?.template?.spec?.nodeSel
 export const getTolerations = (vm: VMKind) => vm?.spec?.template?.spec?.tolerations;
 
 export const getAffinity = (vm: VMKind) => vm?.spec?.template?.spec?.affinity;
+
+export const getDiskCategory = (d: any, volumes: any[]) => {
+  // Check for cdrom category
+  if (d?.cdrom) {
+    return VM_DISK_CATEGORY.CDROM;
+  }
+
+  // Check for disk category
+  if (d?.disk) {
+    const v = volumes.filter((vol) => vol?.name === d?.name)?.[0];
+    if (v?.secret || v?.serviceAccount || v?.configMap) {
+      return VM_DISK_CATEGORY.ENVIRONMENT;
+    }
+
+    return VM_DISK_CATEGORY.DISK;
+  }
+
+  // Other category
```

Comment:
what about luns?

Reply:
+1, only filtering by type now, removed this part of the code.